hips/draw/simple.py

```diff
@@ -10,6 +10,7 @@
 __all__ = [
     'make_sky_image',
     'SimpleTilePainter',
+    'draw_markers_tiles'
```

Comment:
I would suggest to not expose this to end-users, i.e. not list it here.
It's a "hidden feature" that only experts like you and me use.

There's always a tradeoff, what to expose to end-users. Of course it's nice to have everything documented, but then again if the API becomes too large, users will get lost. So it's a judgement call for every function, and in this case, I think we don't need to expose this to end-users.